src/java/picard/analysis/CollectRawWgsMetrics.java

```diff
@@ -5,18 +5,43 @@
 import picard.cmdline.programgroups.Metrics;
 
 /**
- * Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing experiments, same implementation as CollectWgsMetrics, with different defaults: lacks baseQ and mappingQ filters and has much higher coverage cap.
+ * Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing
+ * experiments, same implementation as CollectWgsMetrics, with different defaults: lacks baseQ and mappingQ filters
+ * and has much higher coverage cap.
  *
  * @author farjoun
  */
 @CommandLineProgramProperties(
-        usage = "Computes a number of metrics that are useful for evaluating coverage and performance of " +
-                "whole genome sequencing experiments. Defaults are different than for CollectWgsMetrics.",
-        usageShort = "Writes whole genome sequencing-related metrics for a SAM or BAM file",
+        usage = CollectRawWgsMetrics.USAGE_SUMMARY + CollectRawWgsMetrics.USAGE_DETAILS,
+        usageShort = CollectRawWgsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectRawWgsMetrics extends CollectWgsMetrics{
-
+    static final String USAGE_SUMMARY = "Writes whole genome sequencing-related metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "Useful metrics for evaluating coverage and performance of whole " +
+            "genome sequencing experiments.  Although similar to the CollectWgsMetrics tool," +
+            " the default thresholds for CollectRawWgsMetrics are less stringent.  For example, the CollectRawWgsMetrics" +
+            " has lower base and mapping quality score thresholds as well a higher coverage cap than the CollectWgsMetrics" +
+            " tool.<br /><br />" +
```

Comment:
This description is couched entirely in reference to CollectWgsMetrics, and doesn't say what this tool actually does.  Even if it repeats info from CollectWgsMetrics, I think this explanation should be expanded so it is sufficient to stand on its own.